database/copy.js

```diff
@@ -0,0 +1,39 @@
+//how to run schema: \i /Users/kianna/Desktop/review-module/schema.sql
+const pg = require('pg');
+const connectionString = 'postgres://localhost:5432/project_nomad_reviews';
+
+/*************************** CONNECT TO POSTGRES *****************************/
+const client = new pg.Client(connectionString);
+
+client.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('you have succeeded in seeding');
```

Comment:
you have succeeded in seeding sounds like you have seeded your database, but you really just connected to the db.

'you have connected to postgresDB'